src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -977,6 +978,12 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists(YamlEncoder::class)) {
```

Comment:
In fact no. The class constant has a special behavior.

```<?php

use Something\That\Not\Exist;

echo Exist::class;
```

Will display the FQN even if the class not exist.